integration-test/src/test/java/org/apache/iotdb/pipe/it/autocreate/IoTDBPipeAlterIT.java

```diff
@@ -335,17 +487,26 @@ public void testAlterPipeProcessor() {
     expectedResSet.add("2000,3.0,");
     expectedResSet.add("3000,5.0,");
     TestUtils.assertDataEventuallyOnEnv(
-        receiverEnv, "select * from root.**", "Time,root.db.d1.at1,", expectedResSet);
+        receiverEnv, "select * from root.db.**", "Time,root.db.d1.at1,", expectedResSet);
 
-    // Alter pipe (modify 'processor.tumbling-time.interval-seconds')
+    // Alter pipe (modify 'source.path' and 'processor.tumbling-time.interval-seconds')
     try (final Connection connection = senderEnv.getConnection();
         final Statement statement = connection.createStatement()) {
       statement.execute(
-          "alter pipe a2b modify processor ('processor.tumbling-time.interval-seconds'='2')");
+          "alter pipe a2b modify source('source' = 'iotdb-source','source.path'='root.db.d2.**') modify processor ('processor.tumbling-time.interval-seconds'='2')");
     } catch (SQLException e) {
       fail(e.getMessage());
     }
 
+    // Insert data on sender
+    if (!TestUtils.tryExecuteNonQueriesWithRetry(
+        senderEnv,
+        Arrays.asList(
+            "insert into root.db.d2 (time, at1) values (11000, 1), (11500, 2), (12000, 3), (12500, 4), (13000, 5)",
```

Comment:
Also insert into root.db.d1.at1.

Reply:
Already added, here it is [https://github.com/apache/iotdb/pull/12932/files/342527018d1153a3f604e2c74fa2769ef31835f9#diff-f661440b70bc17d2d84b1a467dc166684f4e9923d0db01bd56eb90623ad543cdR513-R517](https://github.com/apache/iotdb/pull/12932/files/342527018d1153a3f604e2c74fa2769ef31835f9#diff-f661440b70bc17d2d84b1a467dc166684f4e9923d0db01bd56eb90623ad543cdR513-R517)